ansible/init.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+# Creates service account for each operator-pipeline env; updates secret-vars.yml with token
+#
+# As arguments, it expects environments for which the sa should be created
+#
+# ./init.sh dev stage prod
+#       creates the sa for dev, stage and prod environments
+# ./init.sh
+#       creates the sa for every environment (prod, stage, qa, dev)
+
+set -euo pipefail
+umask 077
+
+PARAMS="$*"
+
+# parse arguments given to the script
+get_environments() {
+    if [ -z "$PARAMS" ]; then
+        echo "dev stage qa prod"
+    else
+        echo "$PARAMS"
+    fi
+}
+
+# execute playbook for given environment
+execute_playbook() {
+    local secret=$(dirname "$0")/vaults/$env/secret-vars.yml
+    if [ ! -z $secret ]; then
```

Comment:
You can use -n instead of -z:
```
-n STRING | The length of STRING is greater than zero.
-z STRING | The lengh of STRING is zero (ie it is empty).
```
shellcheck told me that, I recommend this tool

Reply:
Btw. What this condition checks? This string is just concatenated, it never has length equal to 0, script always executes
`echo "File $secret was not found, empty one was created"`

Reply:
Good point, I originally copied the script from our other services. I changed it to -f which should be better here.